src/RestfulServer.php

```diff
@@ -297,7 +296,8 @@ protected function getSearchQuery(
         $sort = null,
         $limit = null,
         $existingQuery = null
-    ) {
+    )
+    {
```

Comment:
I'm not sure this is PSR-2 compliant - you can read more at http://www.php-fig.org/psr/psr-2/
running `phpcs` with the psr-2 standard (use the `--standard` flag, and use the file in the module's root) should identify these things for you, making it easy :)

Reply:
Yeah, see here :)
https://travis-ci.org/silverstripe/silverstripe-restfulserver/jobs/312211524#L639